sorts/slowsort.py

```diff
@@ -0,0 +1,61 @@
+"""
+Slowsort is a sorting algorithm. It is of humorous nature and not useful.
+It's based on the principle of multiply and surrender,
+a tongue-in-cheek joke of divide and conquer.
+It was published in 1986 by Andrei Broder and Jorge Stolfi
+in their paper Pessimal Algorithms and Simplexity Analysis
+(a parody of optimal algorithms and complexity analysis).
+
+Source: https://en.wikipedia.org/wiki/Slowsort
+"""
+
+from math import floor
+
+
+def slowsort(A: list, i: int = None, j: int = None) -> None:
```

Comment:
Python's `typing` module has a way of dealing with arguments that default to None. Also, a should be lowercase, as by python convention:

```python
from typing import Optional

def slowsort(a: list, i: Optional[int] = None, j: Optional[int] = None) -> None:
    ...
```

Reply:
I first thought it was too much to use `Typing`, but I guess you're right in including it, so I'll include it, Thanks.